src/lib/cloud-manager-hoc.jsx

```diff
@@ -46,18 +46,22 @@ const cloudManagerHOC = function (WrappedComponent) {
             // This should cover info about the website specifically, like scrather status
             return !!(props.cloudHost && props.username && props.vm && props.projectId);
         }
+        shouldNotModifyCloudData (props) {
```

Comment:
This is not a great name, but writing this function so that the name is not a `negative` check makes the code less readable...